src/commands/functions/utils/isSiteIdValid.ts

```diff
@@ -0,0 +1,14 @@
+import type { FleekSdk } from '@fleek-platform/sdk/node';
+
+export const isSiteIdValid = ({
+  siteId,
+  sdk,
+}: { siteId: string; sdk: FleekSdk }) => {
+  if (!siteId) return true;
```

Comment:
This should be actually false. A falsely of any kind is not a valid site id.

```suggestion
  if (!siteId) return false;
```

Notice that you can remove the `siteId` assertion where this is called:

```ts
if (!await isSiteIdValid({ siteId, sdk })) { ... }
```

Reply:
I'm removing it actually, the value should only be validated if truthy and the parameter isn't nullable